### PR TITLE
Doc updates

### PR DESCRIPTION
**Pull request by timgilbert**
Sadly the Nobel prize for literature has been canceled this year so I'm out of the running for that, but here are some stillsuit documentation updates nonetheless.

### CODE REVIEW

**Comment by timgilbert** [comment]
The last commit there adds more docs and also adds some convenience functions to get a  connection and/or current db value from datomic (which will let us change up the implementation if we need to).

File: doc/manual/index.adoc [hunk 1]
```diff
@@ -69,28 +71,77 @@ it before returning lacinia input data.
```
Comment (bballant) [inline doc/manual/index.adoc]: Sorry, not in your PR, but the "What Sillsuit provides" section above doesn't seem very useful. What is "a bunch of stuff"?

File: doc/manual/index.adoc [hunk 2]
```diff
@@ -69,28 +71,77 @@ it before returning lacinia input data.
 
 stillsuit's public interface is fairly simple. The
 http://docs.workframe.com/stillsuit/current/doc/stillsuit.core.html#var-decorate[`(stillsuit/decorate)`]
-function accepts a stillsuit config, a datomic connection, and some references to resolvers
+function accepts a lacinia configuration map, a datomic connection, and some references to resolvers
 and bits of code. It returns a map containing the parameters you need to pass to lacinia's
 http://walmartlabs.github.io/lacinia/com.walmartlabs.lacinia.html#var-execute[`(lacinia/execute)`]
 function.
```
Comment (bballant) [inline doc/manual/index.adoc]: This is somewhat repetitive.

File: doc/manual/index.adoc [hunk 3]
```diff
@@ -62,35 +64,112 @@ Here's some sample code corresponding to the above diagram:
...
+
+`:stillsuit/compile?` - if set to `false`, stillsuit won't compile the lacinia schema configuration
+returned in the `:stillsuit/schema` key of `(stillsuit/decorate)`.
 
-* config file
-** sample config or link to one
-** loading the config
+(More TBD)
 
 == The `:stillsuit/ref` resolver
```
Comment (bballant) [inline doc/manual/index.adoc]: Not sure if you should say something about how things just work, most of the time, for primitive types. You allude to it below, but I think it needs a little more context.

File: doc/manual/index.adoc [hunk 4]
```diff
@@ -1,14 +1,17 @@
-= Introduction to stillsuit
+= stillsuit user manual
 Tim Gilbert <tim@workframe.com>
-v0.1, 2018-01-26
+v0.10, 2018-05-04
 :sectanchors:
+:toc: left
+:source-highlight: true
 
 `stillsuit` is a library intended to be used with lacinia and datomic.
```
Comment (bballant) [inline doc/manual/index.adoc]: I think you need to do more here to explain the purpose of stillsuit. Not from a high-level "How" perspective, but more from a high-level "What and Why" perspective.

File: doc/manual/index.adoc [hunk 5]
```diff
@@ -236,20 +315,171 @@ a custom resolver.
...
+In stillsuit we use the term "query resolver" to refer to a lacinia resolver that returns one
+or more entities the provide the basic data for a GraphQL query.
+
+Basically a query is a regular resolver function that will return one or more
+https://docs.datomic.com/on-prem/entities.html[datomic entity objects].
+
+Here's an example of writing a query resolver. First let's define the datomic schema we'll
+be working with.
+
+First we'll define the lacinia schema for the query. For this example, we'll define a very simple
```
Comment (bballant) [inline doc/manual/index.adoc]: You repeat "First" here. I think you can be more concise here in the setup and let the code do the talking.

File: doc/manual/index.adoc (hunk 3 above)
Comment (timgilbert) [reply doc/manual/index.adoc]: Yeah, I'm not done with that yet but I wanted to put it in the "default resolver" section.

**Comment by bballant** [comment]
I left a few more random comments and started skimming toward the end when I encountered a few TBD. Overall it is looking good and I am sure it will continue to evolve. I suggest merging it as soon as you're ready 🍔

File: doc/manual/index.adoc [hunk 6]
```diff
@@ -62,35 +64,112 @@ Here's some sample code corresponding to the above diagram:
...
+  those values will override the default values.
+* A map passed as the `:stillsuit/config` key in the options map of `(stillsuit/decorate)` will
+  override both of the above values.
+
+The config settings from all three places are deep-merged.
+
+=== Config settings
+
+`:stillsuit/compile?` - if set to `false`, stillsuit won't compile the lacinia schema configuration
+returned in the `:stillsuit/schema` key of `(stillsuit/decorate)`.
```
[the remark this answers is not in the document]
Comment (timgilbert) [reply doc/manual/index.adoc]: It's a lacinia thing, I'll drop a link to those docs here.

File: doc/manual/index.adoc (hunk 4 above)
Comment (timgilbert) [reply doc/manual/index.adoc]: Added a little more detail here

File: doc/manual/index.adoc (hunk 5 above)
Comment (timgilbert) [reply doc/manual/index.adoc]: I tightened up the prose some.

**Pull request merged**